src/service/user-service.js

```diff
@@ -0,0 +1,83 @@
+const UserModel = require("../models/user-model");
+const bcrypt = require("bcrypt");
+const tokenService = require("../service/token-service");
+const UserDto = require("../dtos/user-dto");
+const ApiError = require("../exceptions/api-error");
+
+class UserService {
+  async registration(login, password) {
+    const candidate = await UserModel.findOne({ login });
+
+    if (candidate) {
+      throw ApiError.BadRequest(`Пользователь ${login} уже существует`);
+    }
+
+    const hashPassword = await bcrypt.hash(password, 3);
+
```

Comment:
Are you doing too many indents? Why do you separate almost every line with an empty line?